src/sdl_milton.cc

```diff
@@ -344,6 +344,10 @@ sdl_event_loop(Milton* milton, PlatformState* platform)
                     if ( event.button.button == SDL_BUTTON_MIDDLE ) {
                         platform->is_middle_button_down = false;
                     }
+                    if ( ImGui::GetIO().WantCaptureMouse ) {
```

Comment:
Yes, the gui didn't update after button clicks. This was hidden because it was updating every frame after any gui click.
However, I don't know what WantCaptureMouse means, I just copied it from mouse-down handler.

Reply:
I think Milton should force update after any imgui event (click, drag, ...) because it might change data that is used to determine the gui. But I don't know if such flag exists in imgui.